src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -15710,6 +15710,11 @@ void WriteLoadComponentSummaryTables(EnergyPlusData &state)
                         airLoopCoolTable.diffDesignPeak = airLoopHeatTable.designPeakLoad - airLoopHeatTable.peakDesSensLoad;
                     }
                 }
+
+                // Coincident and NonCoincident alike
+                airLoopCoolTable.mixAirTemp = finalSysSizing.MixTempAtCoolPeak;
+                airLoopHeatTable.mixAirTemp = finalSysSizing.HeatMixTemp;
+
```

Comment:
Get the entries. The ZoneSizingData struct does not store the airloop's mixed air temperature, so we just get the value for coincident and noncoincident in the same way, by retrieving it on the FinalSysSizing entry.

Reply:
This is set inside the loop 1 to n for air loops. Seems perfectly reasonable to me.